_posts/2015-09-17-modern-blog.markdown

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: "ModernBlog"
```

Comment:
Thanks for the fixes, also please remove the quotation marks, makes reading plaintext files easier.